### PR TITLE
Fix Korn toolinfo module, include data_model

### PR DESCRIPTION
**Pull request by gernst**

### CODE REVIEW

File: benchexec/tools/korn.py [hunk 1]
```diff
@@ -34,8 +37,21 @@ def version(self, executable):
...
+
+        data_model = task.options.get("data_model")
+
+        if data_model == "ILP32":
+            cmd = cmd + ["-32"]
+
+        if data_model == "LP64":
+            cmd = cmd + ["-64"]
+
+        cmd = cmd + [task.single_input_file]
```
Comment (PhilippWendler) [inline benchexec/tools/korn.py:52]: Compared to the previous implementation, this restricts the tool-info module to tasks with exactly one input file. But I guess that the tool does not support other tasks anyway?

File: benchexec/tools/korn.py [hunk 2]
```diff
@@ -34,8 +37,21 @@ def version(self, executable):
     def name(self):
         return "Korn"
 
-    def project_url(self):
-        return "https://github.com/gernst/korn"
```
Comment (PhilippWendler) [inline benchexec/tools/korn.py:38]: Let's keep this? Then you can remove it from the comment again.

File: benchexec/tools/korn.py [hunk 3]
```diff
@@ -34,8 +37,21 @@ def version(self, executable):
...
+        cmd = [executable]
+        cmd = cmd + options
+
+        data_model = task.options.get("data_model")
+
+        if data_model == "ILP32":
+            cmd = cmd + ["-32"]
+
+        if data_model == "LP64":
+            cmd = cmd + ["-64"]
```
Comment (PhilippWendler) [inline benchexec/tools/korn.py:50]: Many tool-info modules check whether the option is already present, and if yes, do not add it again. This keeps old benchmark definitions working. Not crucial, but maybe nice to have?

There is also a utility function in `sv_benchmarks_util`, which you could use, for example like this:
```python
        data_model_param = get_data_model_from_task(
            task, {ILP32: "-32", LP64: "-64"}
        )

        if data_model_param and data_model_param not in options:
            cmd += data_model_param
```

Comment (gernst) [reply benchexec/tools/korn.py]: Thanks! Sounds good.

File: benchexec/tools/korn.py (hunk 2 above)
Comment (gernst) [reply benchexec/tools/korn.py]: oh right

File: benchexec/tools/korn.py (hunk 1 above)
Comment (gernst) [reply benchexec/tools/korn.py:51]: Yes, that is intended

**Review by PhilippWendler — APPROVED** [review]

**Comment by PhilippWendler** [comment]
In the future, please use proper commit messages. For now I am just going to squash the commits.

**Pull request merged**